llama-index-integrations/llms/llama-index-llms-openai-like/llama_index/llms/openai_like/utils.py

```diff
@@ -0,0 +1,12 @@
+from types import MappingProxyType
+from typing import Any, Dict
+
+# Use these as kwargs for OpenAILike to connect to LocalAIs
+DEFAULT_LOCALAI_PORT = 8080
```

Comment:
Want to change this to OPENAI_LIKE_DEFAULTS but ideally these are LocalAI defaults

confused on this 🤯 , any suggestion would be great!

Reply:
I'm thinking can we have a file for constants in `llama-index-integrations`  to put constants like the above one for easy access in all the third party LLMs?

Reply:
I think tbh localai is deprecated in favor of openai-like. I can't remember if the localai package still exists or if it was aliased to openai-like lol

Reply:
The package exist for LocalAI: https://pypi.org/project/llama-index-llms-localai/
So the doc should also be updated:https://docs.llamaindex.ai/en/stable/examples/llm/localai.html#llamaindex-interaction 
as it points towards LocalAI but uses OpenAILike 😄  class to interaction.

Reply:
What should be the course of action on this @logan-markewich  then if localAI is going to be deprecated!.

Reply:
Yeaaa I think localai mistakingly got made into a package lol. I saw we deal with it in another PR 😅

Reply:
Ah okay, so if you look at the changes I made in this PR is actually removing dependency from localai such as importing default values from localai llm package